RocketDataTests/CollectionDataProviderTests/SharedCollectionDataProviderTests.swift

```diff
@@ -790,6 +791,7 @@ class SharedCollectionDataProviderTests: SharedCollectionTests {
             finishCacheLoad = {
                 completion(initialModels, nil)
             }
```

Comment:
up to you. I think it is perfectly fine to declare `initialModels` outside the scope of the block.

It is no unusual to setup your mock data first and then inject it later for testing.